envtpl.py

```diff
@@ -41,6 +41,11 @@ def main():
                         'with "%s", the output filename is the same as the input '
                         'filename, sans the %s extension. Otherwise, defaults to stdout.' %
                         (EXTENSION, EXTENSION))
+    parser.add_argument('--var-delimiter', default='{{,}}',
```

Comment:
Prefer full words instead of abbreviations: `--variable-delimiter`.

Reply:
Could there be a short option too or an envvar?  This could make for some looong command lines :)

May I suggest that a space could read nicer than a comma?  The value will need to be quoted for the shell anyway (common delimiters are `{ }`, `${ }`, `{{ }}`, `( )`, `< >`) and IMHO it is nicer to read with a space.

(After parsing, I’d argue the internal representation (i.e. the values send to the `_render` function) should be two separate strings or a tuple with two strings)